bin/phase-functions.sh

```diff
@@ -489,7 +489,9 @@ __dyn_test() {
 		die "The source directory '${S}' doesn't exist"
 	fi
 
-	if has test ${RESTRICT} ; then
+	# elide test restriction if the ebuild has PROPERTIES=network-test
+	# and the user enabled FEATURES=network-test
+	if has test ${RESTRICT} && ! { has network-test ${PROPERTIES} && has network-test ${FEATURES}; }; then
```

Comment:
It looks like we'll have to evaluate USE conditionals in PROPERTIES prior to this. See this related RESTRICT code which has no PROPERTIES counterpart:

https://github.com/gentoo/portage/blob/portage-3.0.18/lib/portage/package/ebuild/config.py#L1757
https://github.com/gentoo/portage/blob/portage-3.0.18/bin/ebuild.sh#L621

Reply:
Done!